src/atc/config_master/table_configurator.py

```diff
@@ -10,64 +11,125 @@
 from atc.config_master.exceptions import UnknownShapeException
 from atc.singleton import Singleton
 
+from ..atc_exceptions import AtcException
 from .configuration_types import (
     AnyDetails,
     TableAlias_keys,
-    TableDetailsExtended,
     TableDetailsExtended_keys,
-    TableRelDbg,
     TableRelDbg_keys,
 )
 
 
+class NoSuchPropertyException(AtcException):
```

Comment:
put into atc exceptions.py

Reply:
I disagree, this should only be done once it is used elsewhere.